test/units/module_utils/urls/test_parsing.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# (c) 2018 Matt Martz <matt@sivel.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible.module_utils.urls import ParseResultDottedDict, generic_urlparse
+from ansible.module_utils.six.moves.urllib.parse import urlparse, urlunparse
+
+
+def test_ParseResultDottedDict():
```

Comment:
Move this test function into a new `test_ParseResultDottedDict.py` file to match the class it is testing and give the test function `test_ParseResultDottedDict` a more specific name. Alternatively, move the test into test_urls.py instead.